hazelcast/src/main/java/com/hazelcast/flakeidgen/impl/FlakeIdGeneratorProxy.java

```diff
@@ -113,6 +113,7 @@ public IdBatch newIdBatch(int batchSize) {
 
     @Override
     public long newId() {
+        getService().incrementNewId(name);
```

Comment:
Does not make send to increment here: this would not count IDs generated on client.
You should do it in `newIdBatchLocal` method.

Reply:
Thx. I moved the inc to that method. I still only increment by 1 and not by the batch size because I think this number is more useful for the statistics we want to do than mixing it up with the batch size.